coordinator/src/main/scala/com/socrata/datacoordinator/primary/MigrateSchema.scala

```diff
@@ -37,5 +37,6 @@ object MigrateSchema extends App {
 
     SchemaMigrator(databaseTree, operation, numChanges)
   }
-  private lazy val databaseTree = "com.socrata.coordinator.common.database"
+
+  private lazy val databaseTree = s"${com.socrata.datacoordinator.service.Main.configRoot}.database"
```

Comment:
It is more like a constant.  Is it the name, being a val or relating the two classes that concerns you?

There are alternatives like
1. repeat the same string literal.
2. change the config file.
3. make Make.make run the migration class by detecting some argument.
and more...